app/src/main/java/com/powerhungers/fomezero/feature/login/presentation/LoginFragment.kt

```diff
@@ -46,9 +47,14 @@ class LoginFragment : Fragment() {
         viewModel.loginLiveData.observe(viewLifecycleOwner) { state ->
             with(binding) {
                 when (state) {
-                    is ViewState.Loading -> Unit
-                    is ViewState.Success -> Unit
-                    is ViewState.Error -> Unit
+                    is ViewState.Loading ->
+                        progressDialog.visibility = View.VISIBLE
+                    is ViewState.Success -> {
+                        findNavController().navigate(
+                            LoginFragmentDirections.actionLoginFragmentToOnboardFragment())
```

Comment:
```suggestion
                            LoginFragmentDirections.actionLoginFragmentToOnboardingFragment())
```
geralmente chamamos esse tipo de tela de Onboarding com ing no final